sphinx_scality/static/js/searchtools.js

```diff
@@ -0,0 +1,508 @@
+/*
+ * searchtools.js
+ * ~~~~~~~~~~~~~~~~
+ *
+ * Sphinx JavaScript utilities for the full-text search.
```

Comment:
Let's split the addition of this `searchtools.js` file from the customization (in terms of Git commits), it will help with identifying what changes were introduced. Another approach could have been to import this JS module from another one we own and only override the methods we need, but that may be harder to implement.